docs/cvl/methods.md

```diff
@@ -39,7 +39,10 @@ methods          ::= "methods" "{" { method_spec } "}"
 
 method_spec      ::= ( sighash | [ id "." ] id "(" evm_params ")" )
                      [ "returns" types ]
-                     [ "envfree" ]
+                     { | "with" "(" "env" id ")"
```

Comment:
Do we want the `with` on the left-hand side of the `=>`? or maybe on the right-hand side, similar to the `expects` clause?

Reply:
I moved it to the right side.